pkg/operator/ceph/cluster/osd/osd.go

```diff
@@ -43,15 +44,16 @@ import (
 var logger = capnslog.NewPackageLogger("github.com/rook/rook", "op-osd")
 
 const (
-	appName                      = "rook-ceph-osd"
-	prepareAppName               = "rook-ceph-osd-prepare"
-	prepareAppNameFmt            = "rook-ceph-osd-prepare-%s"
-	legacyAppNameFmt             = "rook-ceph-osd-id-%d"
-	osdAppNameFmt                = "rook-ceph-osd-%d"
-	osdLabelKey                  = "ceph-osd-id"
-	clusterAvailableSpaceReserve = 0.05
-	serviceAccountName           = "rook-ceph-osd"
-	unknownID                    = -1
+	appName                             = "rook-ceph-osd"
+	prepareAppName                      = "rook-ceph-osd-prepare"
+	prepareAppNameFmt                   = "rook-ceph-osd-prepare-%s"
+	legacyAppNameFmt                    = "rook-ceph-osd-id-%d"
+	osdAppNameFmt                       = "rook-ceph-osd-%d"
+	osdLabelKey                         = "ceph-osd-id"
+	clusterAvailableSpaceReserve        = 0.05
+	serviceAccountName                  = "rook-ceph-osd"
+	unknownID                           = -1
+	cephOsdPodMinimumMemory      uint64 = 4096 // minimum amount of memory in MB to run the pod
```

Comment:
I just realized now that this value has not been documented as users are hitting this "issue" (`2019-05-14 09:40:15.256685 E | op-cluster: failed to create cluster in namespace rook-ceph. failed to start the osds. refuse to run the pod with 1024mb of ram, provide at least 4096mb.`)..

@leseb could you please open a PR to add a note to the docs about this minimum and backport it to v1.0 too. If you are unavailable, I'm happy to create a PR too :-)

Reply:
Fixed in https://github.com/rook/rook/pull/3258, thanks!